sanctions/apps/sanctions/models.py

```diff
@@ -0,0 +1,84 @@
+from simple_history.models import HistoricalRecords
+
+from django.core.validators import MinLengthValidator
+from django.db import models
+from django_extensions.db.models import TimeStampedModel
+from django.utils.translation import gettext_lazy as _
+
+class SanctionsCheckFailure(TimeStampedModel):
+    """ Record of SDN and ISN check failure. """
+    history = HistoricalRecords()
+    full_name = models.CharField(max_length=255)
+    username = models.CharField(max_length=255)
+    lms_user_id = models.IntegerField(null=True, db_index=True)
+    city = models.CharField(max_length=32, default='')
+    country = models.CharField(max_length=2)
+    sanctions_type = models.CharField(max_length=255)
+    system_identifier = models.CharField(max_length=255)
```

Comment:
This would be which system/service the request came from, in case in the future we use the sanctions check endpoint from other things like subscriptions

Reply:
interesting. was this for a requirement you were made aware of? or just a nice to have? i'm not necessarily opposed to it, just curious

Reply:
Not a requirement